.github/workflows/build.yml

```diff
@@ -1035,3 +1109,232 @@ jobs:
+            container: node:16-alpine3.16
+            node: 16
+          - os: ubuntu-20.04
+            container: node:18-alpine3.17
+            node: 18
+          - os: ubuntu-20.04
+            container: node:20-alpine3.17
+            node: 20
+
+            # arm64 glibc
+          - os: ubuntu-20.04
+            arch: arm64
+            node: 16
+          - os: ubuntu-20.04
+            arch: arm64
+            node: 18
+          - os: ubuntu-20.04
+            arch: arm64
+            node: 20
+
+            # arm64 musl
+          - os: ubuntu-20.04
+            container: node:16-alpine3.16
+            arch: arm64
+            node: 16
+          - os: ubuntu-20.04
+            arch: arm64
+            container: node:18-alpine3.17
+            node: 18
+          - os: ubuntu-20.04
+            arch: arm64
+            container: node:20-alpine3.17
+            node: 20
+
+            # macos x64
+          - os: macos-11
+            node: 16
+            arch: x64
+          - os: macos-11
+            node: 18
+            arch: x64
+          - os: macos-11
+            node: 20
+            arch: x64
+
+            # macos arm64
+          - os: macos-12
+            arch: arm64
+            node: 16
+            target_platform: darwin
+
+          - os: macos-12
+            arch: arm64
+            node: 18
+            target_platform: darwin
+
+          - os: macos-12
+            arch: arm64
+            node: 20
+            target_platform: darwin
+
+            # windows x64
+          - os: windows-2022
+            node: 16
+            arch: x64
+
+          - os: windows-2022
+            node: 18
+            arch: x64
+
+          - os: windows-2022
+            node: 20
+            arch: x64
+    steps:
+      - name: Setup (alpine)
+        if: contains(matrix.container, 'alpine')
+        run: |
+          apk add --no-cache build-base git g++ make curl python3
+          ln -sf python3 /usr/bin/python
+
+      - name: Check out current commit (${{ needs.job_get_metadata.outputs.commit_label }})
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ env.HEAD_COMMIT }}
+
+      - name: Restore dependency cache
+        uses: actions/cache/restore@v3
+        id: restore-dependencies
+        with:
+          path: ${{ env.CACHED_DEPENDENCY_PATHS }}
+          key: ${{ needs.job_install_deps.outputs.dependency_cache_key }}
+          enableCrossOsArchive: true
+
+      - name: Restore build cache
+        uses: actions/cache/restore@v3
+        id: restore-build
+        with:
+          path: ${{ env.CACHED_BUILD_PATHS }}
+          key: ${{ needs.job_build.outputs.dependency_cache_key }}
+          enableCrossOsArchive: true
+
+      - name: Configure safe directory
+        run: |
+          git config --global --add safe.directory "*"
+
+      - name: Install yarn
+        run: |
+          npm i -g yarn@1.22.19 --force
+          yarn config set network-timeout 600000 -g
```

Comment:
Needed to bump yarn network timeout as windows is slow

Reply:
😢 

Can we only do the timeout on windows?

Reply:
Yes of course!